server/athenian/api/precompute/discover_accounts.py

```diff
@@ -28,5 +37,30 @@ async def main(context: PrecomputeContext, args: argparse.Namespace) -> List[int
         if state.finished_date is None:
             log.warning("Skipped account %d because the progress is not 100%%", account)
             continue
-        result.append(account)
-    return result
+
+        discovered["precomputed" if precomputed else "fresh"].append(account)
+
+    if args.partition:
+        return discovered
+    else:
+        return sorted(chain(*discovered.values()))
+
+
+async def _get_accounts(sdb: Database) -> List[Tuple[int, bool]]:
+    """Return the existing account IDs, each with a precomputed flag."""
+    # already precomputed accounts have an ALL repository set that is precomputed
+    # outer join will return one row per account, since (owner_id, name) is unique in RepositorySet
+    join_cond = sa.and_(
+        Account.id == RepositorySet.owner_id,
+        RepositorySet.name == RepositorySet.ALL,
+    )
+    stmt = sa.select(
+        [Account.id, sa.func.coalesce(RepositorySet.precomputed, False).label("precomputed")],
```

Comment:
`RepositorySet.precomputed` is `nullable=False`, we have a strong guarantee here.